README.md

```diff
@@ -1344,10 +1344,31 @@ async def check_past_histories(my_client: Client):
   )
 ```
 
-### OpenTelemetry Support
+### Observability
 
-OpenTelemetry support requires the optional `opentelemetry` dependencies which are part of the `opentelemetry` extra.
-When using `pip`, running
+See https://github.com/temporalio/samples-python/tree/main/open_telemetry for a sample demonstrating collection of
+metrics and tracing data emitted by the SDK.
+
+#### Metrics
+
+The SDK emits various metrics by default: see https://docs.temporal.io/references/sdk-metrics. By default, these are
+emitted with attributes `namespace`, `task_queue`, `workflow_type` / `activity_type`, and
+`service_name=temporal-core-sdk`. To emit additional attributes with all metrics, pass
```

Comment:
This is not accurate. We emit lots of client side metrics, worker metrics, etc that don't have some (or all) of these attributes.

Reply:
Thanks, deleted